blogpost/blogpost.md

```diff
@@ -0,0 +1,845 @@
+# Apache Beam streaming in a Spring Boot application. Use case.
+In this blog post I'll write a simple Spring Boot application using Apache Beam to stream data (with Apache Flink under the hood) from Apache Kafka to MongoDB and expose endpoints providing real-time data. The application will simulate a data center that receives data about lightnings around the world from a Kafka instance.
+
+It will expose 2 endpoints: one that returns up-to-date data about received lightnings and the second one that returns how many lightnings stroke the ground in one minute time range.
+
+I will also write an emulator that will simulate the detectors that receive the lightning and send the data to a Kafka instance.
+
+All the data processing will be executed via Apache Beam with Apache Flink as the runner. The endpoints will use Spring Data repositories which is a super convenient tool for this NoSQL database integration.
+
+If you are not familiar with Spring Boot I encourage you to read Michal's great [Spring Boot article](https://www.polidea.com/blog/what-is-spring-boot-first-steps-with-spring/). To learn more about Apache Beam I recommend Kamil's [Apache Beam Tutorial](https://www.polidea.com/blog/apache-beam-tutorial-and-beginners-guide/) and [Apache Beam Quickstart](https://beam.apache.org/get-started/quickstart-java/).
+
+## Setting things up
+
+At first I want to use Spring Boot. I will use Spring Initializr so I can easily start with a ready-to-go project.
+
+![image](initializr.png)
+
+I usually use Gradle but for this simple project I've chosen Maven as the build tool - it's popular and simple. I'll not do any magic here so it's perfectly sufficient.
+
+Next, I checked Spring Boot version 2.2.11 because the Apache Beam's MongoDB IO uses Mongo Java driver 3. The later versions of Spring Data MongoDB use Mongo Java Driver 4 so for now they are not compatible.
```

Comment:
`Later versions of...` lub `The latest version of ...` - w zależności, czy masz na myśli kilka najnowszych wersji, czy jedną najnowszą

Reply:
Dzięki! 👍